### ISSUE TITLE
Geo search: add bbox query

### ISSUE THREAD

**Issue by cnrdh** [npolar/icelastic#43]
Use
http://www.opensearch.org/Specifications/OpenSearch/Extensions/Geo/1.0/Draft_1#The_.22box.22_parameter

**Comment by cnrdh** [comment]
Sure thing:
* https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-geo-shape-type.html
* https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-geo-shape-filter.html
* https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-geo-shape-query.html

**Comment by cnrdh** [comment]
*https://www.elastic.co/guide/en/elasticsearch/reference/1.7/query-dsl-geo-shape-query.html
*https://www.elastic.co/guide/en/elasticsearch/reference/1.7/query-dsl-geo-shape-filter.html

**Comment by cnrdh** [comment]
```json
{
  "query": {
    "filtered": {
      "query": {
        "match_all": {}
      },
      "filter": {
        "geo_shape": {
          "geometries": {
            "shape": {
              "type": "envelope",
              "coordinates": [
                [
                  -180,
                  77
                ],
                [
                  180,
                  78
                ]
              ]
            }
          }
        }
      }
    }
  }
}
```

**Comment by cnrdh** [comment]
Implemented in dec9ca02f9ac326ff465e67fa1bfd6c4ad92dccc
